### DISCUSSION TITLE
Store the camera when leaving the vehicle

### DISCUSSION THREAD

**Discussion by Kynuska** [Mogli12/KeyboardSteer#783] [Ideas]
Hello.
I would like to discuss whether it would be possible to add the camera storage function when leaving the vehicle. (indoor / outdoor).
When used with manual connection, it's annoying to constantly switch into the vehicle when boarding.
there was a script for it in FS19, I didn't find it for FS22.
Best regards.
Kynuska.

**Comment by Dzi4d3k** [comment]
Hi @Kynuska seems you meant:
https://github.com/DeckerMMIV/FarmSim_Mod_QuickCamera
that is still "in development" for FS22

**Comment by Kynuska** [comment]
That probably won't be it.
I don't know if it doesn't include this feature either, I don't know this mod.
Views include VCA too. So that it is not in collision.
When I have an indoor view and get out, I get in the vehicle and the default setting is outdoor.
Kynuska

**Comment by Kynuska** [comment]
probably this mod would solve that.
https://github.com/Stephan-S/FS19_IndoorCamFirst
I will write to the author if he has a version for FS22 planned.
Kynuska
